packages/mui-material/src/Alert/Alert.js

```diff
@@ -71,7 +72,10 @@ const AlertRoot = styled(Paper, {
       }),
     ...(color &&
       ownerState.variant === 'filled' && {
-        color: '#fff',
+        color:
```

Comment:
```suggestion
        color: theme.palette.getContrastText(theme.palette.mode === 'dark' ? theme.palette[color].dark : theme.palette[color].main),
```

Shouldn't it be like this?